ecs-cli/integ/cmd/compose.go

```diff
@@ -85,6 +85,7 @@ func TestServiceUp(t *testing.T, p *Project) {
 		"up",
 		"--cluster-config",
 		p.ConfigName,
+		"--create-log-groups",
```

Comment:
nit: Just a general comment for best practices. I normally try to limit the number of changes in a single PR such that one PR represents a self-contained, logical set of changes. For this PR, that means creating a new PR for the bug fixes, merge it in, then continue working on this PR but keep it strictly for the feature you are working on.

By doing so, it has the following benefits:

1. In case we accidentally introduce any bug in the feature once it's merged, we can just revert the feature without reverting the bug fixes
2. If an issue starts showing up after a certain PR is merged, then it's easier to pinpoint the issue.

Reply:
Good advice! Next time when I encounter problems of another fields, I will create a new branch a submit another PR. Thanks for the advice and explanation.